packages/atlas/src/hooks/useHidePasswordInInput.tsx

```diff
@@ -3,18 +3,20 @@ import { useState } from 'react'
 import { SvgActionHide, SvgActionShow } from '@/assets/icons'
 import { InputProps } from '@/components/_inputs/Input'
 
-export const useHidePasswordInInput = (): [InputProps, () => void] => {
+export const useHidePasswordInInput = (isNew = true): [InputProps, () => void, boolean] => {
   const [isPasswordVisible, setIsPasswordVisible] = useState(false)
 
   return [
     {
       type: isPasswordVisible ? 'text' : 'password',
+      autoComplete: isPasswordVisible ? 'off' : isNew ? 'new-password' : 'off',
```

Comment:
the thing is that default password generator (at least in chrome) doesn't pass our password rules, so I think we can just use the 'off' value here

Reply:
resolved on the call